plugins/scaffolder/src/components/fields/RepoBranchPicker/RepoBranchPicker.test.tsx

```diff
@@ -333,4 +335,56 @@ describe('RepoBranchPicker', () => {
       expect(getByText('abc123')).toBeInTheDocument();
     });
   });
+
+  describe('RepoBranchPicker description', () => {
+    const customDescription = 'Custom RepoBranchPicker description';
+
+    it.each([
```

Comment:
I'm a little torn on the use of each in tests.

It makes the tests really hard to understand without a basic description as to what we're really testing. Could we refactor these into explicit tests with a `it('description')` and what's it's testing?

Reply:
you're the boss